Task5.2/Catalog.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Task52
+{
+    public class Catalog
+    {
```

Comment:
Better to create ISBN class, move normalization and format check to it, and use  Dictionary<ISBN, Book> books